### ISSUE TITLE
Make the tiny-css styling more appealing

### ISSUE THREAD

**Issue by FotieMConstant** [tiny-css/tiny-css#7]
I think the current styling is too plain and not really appealing for a UI

I suggest we rethink the various stylings and bring in some uniformity- Like appealing buttons, input fields, dropdown menus, etc.

Current styling 

![image](https://user-images.githubusercontent.com/42372656/99941303-17918f80-2d6e-11eb-81f5-8ef1dba5a182.png)

I think we can make this more appealing and cooler

**Comment by KRTirtho** [comment]
Of course, I was thinking to change them a bit too. I started this when I only knew bootstrap & jQuery.
But we've to prior our biggest feature which is built-in **Dark-Mode** support  as no other library supports this quite nicely yet. Also have to take care of awesome design prospects.
I suggest using some kind of designer tool (Adobe XD or Figma) first to design the components or default stylings. Then write the css relative to those design sheets. Adobe XD has a great feature called DSP (Design System Package)  which is used by VSCode's **Adobe XD**  extension and it always suggests defined styles or code snippets on CSS files or other files like JSX if defined in the DSP yml file.

Youtube tutorial for DSP: https://www.youtube.com/watch?v=5MhfwvlGkfA,
DSP Github: https://github.com/AdobeXD/design-system-package-dsp
